### ISSUE TITLE
[1.16.1] Zombified piglin aggro not spreading

### ISSUE THREAD

**Issue by dhoell** [PaperMC/Paper#3836]
### What behaviour is expected:
When attacking a zombiefied piglin in a gold/XP farm (circluar model, killing chamber with player in the middle) for the aggro to spread and all piglins to walk towards the player.

### What behaviour is observed:
Only the attacked piglin starts to walk towards the player. It also frequently drops aggro.

### Steps/models to reproduce:
Farm build similar to: https://www.youtube.com/watch?v=s6z5GLcqbIM
Floors made out of magma blocks, to inhibit non-zombie piglin spawning, making trapdoors unnessecary.
Note: Farm is working as intentended on build 68 of paper (same plugins).

### Plugin list:
ChestSort
EssentialsX 2.17.2.146
Login Security 3.0.2
Timber 1.0.7
WorldEdit 7.2.0-beta1

### Paper build number:
74

### Anything else:
Will test builds 69-73 to narrow down the issue as soon as all players on my server are offline.

**Comment by colinthesealion** [comment]
I've seen this on my world as well. Oddly, restarting the server will fix the issue temporarily.

**Comment by Mendanbar** [comment]
I've noticed this as well and thought it might be related to the fact that I set `per-player-mob-spawns` to true.  But it sounds like that might be a red herring? I can't recall which version we first started seeing it in, but it's still happening as of 135.  I'm using [this design](https://www.youtube.com/watch?v=nMBYnGD8PbQ) by Ray's Works, and it was fine for the early 1.16 builds of paper (but as I mentioned I did have per player mob spawns off at the time)
